src/lib/Components/Consignments/Screens/SelectFromPhotoLibrary.tsx

```diff
@@ -80,22 +80,7 @@ export default class SelectFromPhotoLibrary extends React.Component<Props, State
       return
     }
 
-    CameraRoll.getPhotos(fetchParams).then(data => {
-      if (data && data.edges[0] && data.edges[0].node) {
-        const photo = data.edges[0].node
-
-        // Update selection
-        this.state.selection.add(photo.image.uri)
-        this.setState({
-          selection: this.state.selection,
-          // An item in cameraImage is a subset of the photo that we pass in,
-          // so we `as any` to avoid a compiler error
-          cameraImages: data.edges.map(e => e.node).concat(this.state.cameraImages as any),
-        })
-      } else {
-        console.log("CameraRoll: Did not receive a photo from call to getPhotos")
-      }
-    })
+    CameraRoll.getPhotos(fetchParams).then(data => this.appendAssets(data))
```

Comment:
I tried `CameraRoll.getPhotos(fetchParams).then(this.appendAssets)` but `this` gets all screwey.